<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAskEncryptMethod</class>
 <widget class="QDialog" name="DlgAskEncryptMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>112</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>178</width>
    <height>112</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>178</width>
    <height>112</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose encrypt method</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_ptrEncryptMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
